libs/utils/utils.js

```diff
@@ -248,8 +248,12 @@ export function appendHtmlPostfix(area = document) {
 
   const HAS_EXTENSION = /\..*$/;
   const shouldNotConvert = (href) => {
+    let url;
```

Comment:
just in case `new URL` fails, should we init this href?
```suggestion
    let url = { pathname: href };
```